scripts/linters/css_linter.py

```diff
@@ -58,6 +59,36 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from stylelint error messages.
+
+        Args:
+            lint_messages: str. Messages returned by the css linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # We need to extract messages from the list and split them line by
+        # line so we can loop through them.
+        messages = lint_messages.split('\n')
+        for message in messages:
+            # Stylelint messages start with line numbers and then a
+            # cross(x) and a message-id in the end. We are matching
+            # if the line contains line number becuase every message start with
+            # num:num and we are matching it with regex and if that is True then
+            # we are removing cross(x) which is at the index 1 and message-id
```

Comment:
Instead of doing that, perhaps include it in the regex and use a capture group to extract the part you're interested in.